### DISCUSSION TITLE
[AfterMapping] Possible to use @Mapper::uses?

### DISCUSSION THREAD

**Discussion by agreedSkiing** [mapstruct/mapstruct#3198] [Q&A] [answered]
In an `@Aftermapping` is it possible to use the included mappers that are defined in the `@Mapper::uses` array? Or do I need to pass them with `@Context` annotation or define them with a `@Inject` in the interface/abstract mapper?

**Comment by filiphr** [comment, marked as answer]
Currently it is not possible to access the classes defined by `Mapper#uses`. I think that we have a request for that. 

The only possible solutions are to pass them via `@Context` or inject them in your abstract mapper (you can't inject in an interface)
